src/storage.rs

```diff
@@ -514,6 +623,8 @@ impl Session {
     /// 
     /// Returns true if there is a public key for the id which can verify the signature
     /// Returns false if there is no public key for the id or if no public key can verify the signature
+    /// 
+    /// ONLY FOR KEY TRANSPARENCY APPLICATION
```

Comment:
Hmmm, is there a better way to add this? Maybe using that rust "feature" thing ive seen in cargo.tomls?